lib/inaturalist_map_server.js

```diff
@@ -60,6 +60,11 @@ InaturalistMapserver.prepareQuery = function( req, callback ) {
   req.elastic_query.fields = InaturalistAPI.defaultMapFields( );
   // requiring mappable=true to be included in map tiles
   req.query.mappable = "true";
+  if ( req.query.captive === "any" ) {
```

Comment:
Well, I want to keep the test, because I want to make sure we maintain that behavior. Default should be captive=false, but the user should be able to ask for captive=true to only show things that are captive, OR captive=any to show observations that are captive or wild. I'll restructure it to get rid of the reference to `any` in the app code, though.